sag_et.par

```diff
@@ -4,195 +4,241 @@
+)|  /|  /|\  (|\  (|)  ||\  (||~  /||\
 d 62 36
-/|)  /|\  (|\  /N\
+/|)  /|\  (|\  /||\
 d 63 37
 s 21
-|)  /|  /|\  (|)  N\  N)  /N\
+|)  /|  /|\  (|)  ||\  ||)  /||\
 n 64 37
-/|) (|\ /N\
+/|)  (|\  /||\
 d 65 38
-/|  |)  /|\  N)  N\  /N\
+/|  |)  /|\  ||)  ||\  /||\
+n 66 39
+)|  )~|  /|  /|)  (|\  ||\  ~||\  (||~  /||\
 d 67 39
-)|(  /|)  (|\  //N  /N\
+)|(  /|)  (|\  //||  /||\
 d 68 40
-|\  /|  /|\  /|)  (|)  N\  /N  /N\
+|\  /|  /|\  (|\  (|)  ||\  /||  /||\
+d 69 40
+/|)  /|\  (|\  /||\
 d 70 41
 s 35
-/|  |\  /|\  (|)  /N  N\  /N\
+/|  |\  /|\  (|)  /||  ||\  /||\
+n 71 42
+|\  )~|  /|  /|\  (|\  (|)  ||\  ~||\  /||  /||\
 d 72 42
-/|  |)  /|\  N)  N\  /N\
+/|  |)  /|\  ||)  ||\  /||\
 d 75 44
-|\  /|  /|\  /|)  (|)  N\  /N  /N\
+|\  /|  /|\  /|)  (|)  ||\  /||  /||\
 d 77 45
-/|  |)  /|\  (|)  N)  N\  /N\
+/|  |)  /|\  (|)  ||)  ||\  /||\
 d 79 46
-/|  |)  /|\  N)  N\  /N\
+/|  |)  /|\  ||)  ||\  /||\
+d 80 47
+s 40
+|)  /|  (|~  /|\  (|)  ~||(  ||\  ||)  /||\
 d 84 49
 s 42
-/|  |)  /|)  (|\  N)  N\  /N\
+/|  |)  /|)  (|\  ||)  ||\  /||\
 d 87 51
-|~  /|  /|~  /|\  (|)  N~  N\  /N~  /N\
+|~  /|  /|~  /|\  (|)  ||~  ||\  /||~  /||\
 d 89 52
-/|  |)  /|)  /|\  (|\  N)  N\  /N\
+/|  |)  /|)  /|\  (|\  ||)  ||\  /||\
 d 94 55
 s 47
-~|(  /|  (|(  /|\  (|)  ~N(  N\  (N(  /N\
+~|(  /|  (|(  /|\  (|)  ~||(  ||\  (||(  /||\
 d 96 56
 s 32
-/|  |~  |\  /|\  /N  /N~  N\  /N\
+/|  |~  |\  /|\  /||  /||~  ||\  /||\
 d 99 58
 s 33
-~|  /|  ~|)  //|  )/|\  )N(  ~~N  N\  )//N  /N\
+~|  /|  ~|)  //|  )/|\  )||(  ~~||  ||\  )//||  /||\
 d 106 62
-)|(  /|  (|  //|  /|\  )N(  )N~  N\  //N  /N\
+)|(  /|  (|  //|  /|\  )||(  )||~  ||\  //||  /||\
 d 108 63
 s 54
-)/|  (|  |)  /|)  (|\  N)  )N~  )N)  /N\
+)/|  (|  |)  /|)  (|\  ||)  )||~  )||)  /||\
 d 111 65
 s 37
-)|(  /|  |\  //|  /|\  (|)  )N(  /N  N\  //N  /N\
+)|(  /|  |\  //|  /|\  (|)  )||(  /||  ||\  //||  /||\
 d 118 69
 s 59
-)|(  /|  |)  //|  /|\  (|)  )N(  N)  N\  //N  /N\
+)|(  /|  |)  //|  /|\  (|)  )||(  ||)  ||\  //||  /||\
 d 120 70
-)/|  (|  |)  /|~  /|\  N~  N)  )N~  )N)  /N\
+)/|  (|  |)  /|~  /|\  ||~  ||)  )||~  )||)  /||\
 d 125 73
-|(  /|  |)  //|  /|)  (|\  )N(  N)  N\  /N)  /N\
+|(  /|  |)  //|  /|)  (|\  )||(  ||)  ||\  /||)  /||\
 d 130 76
-|(  /|  |)  |\  /|)  /|\  (|\  /N  N)  N\  /N)  /N\
+|(  /|  |)  |\  /|)  /|\  (|\  /||  ||)  ||\  /||)  /||\
 d 132 77
 s 66
-~|(  (|  |~  |\  /|)  (|\  /N  /N~  )N~  (N(  /N\
+~|(  (|  |~  |\  /|)  (|\  /||  /||~  )||~  (||(  /||\
 d 142 83
 s 71
-|(  /|  |)  |\  /|)  /|\  (|)  (|\  /N  N)  N\  /N)  /N\
+|(  /|  |)  |\  /|)  /|\  (|)  (|\  /||  ||)  ||\  /||)  /||\
 d 144 84
-~|(  /|  |~  |)  /|~  /|\  N~  N)  /N~  N\  (N(  /N\
+~|(  /|  |~  |)  /|~  /|\  ||~  ||)  /||~  ||\  (||(  /||\
 d 152 89
 s 76
 )|(  ~|(  /|  |\  (|(  //|  /|\  (|)
-  )N(  ~N(  /N  N\  (N(  //N  /N\
+  )||(  ~||(  /||  ||\  (||(  //||  /||\
 d 156 91
 s 52
 ~|(  /|  |~  |)  |\  /|)  (|\
-  /N  N)  /N~  N\  (N(  /N\
+  /||  ||)  /||~  ||\  (||(  /||\
 d 159 93

```

Comment:
159, 166 (and 83 as subset), 198, and 212 were not in the Xenharmonikon article but did appear here in Scala.

Reply:
I've taken a note to include them when we update the Xenharmonikôn article.

Reply:
Thanks.